includes/admin/plugin-updates/class-wc-updates-screen-updates.php

```diff
@@ -37,8 +39,13 @@ public function update_screen_modal() {
 			return;
 		}
 
+		$version_type = Constants::get_constant( 'WC_SSR_PLUGIN_UPDATE_RELEASE_VERSION_TYPE' );
```

Comment:
This constant is currently defined only in `html-admin-page-status-report.php`, I guess this won't be defined in most flows, which means this `get_constant` call is essentially a no-op (unless you are on the SSR page)? 

Why not move this definition to wocommerce.php or skip the constant check?

Reply:
You're right, thanks for noticing. I wrongly assumed this is more global. But then, on the other hand, it specifically mentions SSR, so don't know if it should actually be reused elsewhere. Does it make sense for the version check to be more granular (e.g. to only hide the warnings in SSR, but show them in the plugin screen)? I can't see how that would be useful to anyone, so I'd lean towards pulling this one out of this file to a global file. We need to preserve the name for sake of bw compatibility, though, so not great, but will add docs.

Reply:
Please see 8566a79.